src/assets/stylesheets/layout/_content.scss

```diff
@@ -52,6 +55,13 @@
   &__inner {
     margin: 0 px2rem(16px) px2rem(24px);
     padding-top: px2rem(12px);
+    // Set height to 100vh minus:
+    // * header height (84px)
+    // * footer height (395px)
+    // * top margin of md-main__inner (30px)
+    // * top margin of md-content__inner (24px)
+    //flex-basis: calc(100vh - 84px - 395px - 30px - 24px);
+    flex-basis: calc(100vh - 26.65rem);
```

Comment:
Is there a better way of setting the maximum height of this container without relying on "magic numbers" calculated from the heights of other elements on the page? :thinking:

Reply:
Thanks @inesgomas, it's good to know this kind of thing isn't as strange as I was thinking! :sweat_smile: I'm going to make another commit just to clean this up a bit. :rocket: